src/components/Challenge/EventViewer.tsx

```diff
@@ -26,17 +26,18 @@ const Description = styled('div', {
 interface EventViewerProps extends StyleProps {
   event: Event;
   eventState?: boolean;
+  locale: LocalizedString.Language;
 }
 
-const EventViewer: React.FC<EventViewerProps> = ({ event: { name, description }, eventState, style, className }) => {
+const EventViewer: React.FC<EventViewerProps> = ({ event: { name, description }, eventState, style, className, locale }) => {
   return (
     <Container style={style} className={className}>
       <NameContainer>
-        <Name>{LocalizedString.lookup(name, LocalizedString.EN_US)}</Name>
+        <Name>{LocalizedString.lookup(name, locale)}</Name>
         <Spacer />
         {eventState !== undefined && <FontAwesomeIcon icon={eventState ? faCheck : faCross} />}
       </NameContainer>
-      {description && <Description>{LocalizedString.lookup(description, LocalizedString.EN_US)}</Description>}
+      {description && <Description>{LocalizedString.lookup(description, locale)}</Description>}
     </Container>
   );
 };
```

Comment:
`EventSettings.tsx` has a few hardcoded `EN_US` for event names/descriptions. not sure how those are supposed to be handled

Reply:
EventSettings isn't currently used and isn't finished. I suggest we leave as is.